.github/workflows/test.yml

```diff
@@ -19,6 +19,8 @@ on:
       cylc_repo:
         description: The Cylc repo to test against
         required: false
+  schedule:
+  - cron: '37 04 * * 1-5' # 03:37, Monday-Friday
```

Comment:
I thought the idea was to run once or twice a week? Also the comment says 03:37 whereas it is actually 04:37